### DISCUSSION TITLE
vlang on Github Codespace

### DISCUSSION THREAD

**Discussion by thesombady** [vlang/v#17212] [Questions and Answers]
Hi,

I've recently started learning *V*, due to its versatility!  I enjoy the language extremely for the uses I intend however, I'm wondering whether one can setup such an enviroment on github codespaces, and if so the templete structure. It's not avaiable as a default image nor container.

Wishing the best,

**Comment by JalonSolov** [comment]
Please try https://play.vlang.io - it is "instant" dev for V, and there are no time limits.

**Reply by thesombady** [reply]
Hi,

It's a possibility for small script but for more extensive projects this is not a viable solution. As of right now I do have a local installation of vlang, and can thus run my project without any problem; just curious if it would be possible to do such a project on github codespace.

Best wishes

**Reply by JalonSolov** [reply]
I'm sure it is something that will be done eventually.  I am unaware of anyone working on it right now.
